__tests__/app.test.js

```diff
@@ -61,6 +61,76 @@ describe("GET: /api/reviews/:review_id", () => {
   });
 });
 
+describe("PATCH /api/reviews/:review_id", () => {
+  test("200, responds with the updated review", () => {
+    const reviewUpdate = { inc_votes: 1 };
+    return request(app)
+      .patch("/api/reviews/1")
+      .send(reviewUpdate)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual({
+          updatedReview: {
+            review_id: 1,
+            title: "Agricola",
+            category: "euro game",
+            designer: "Uwe Rosenberg",
+            owner: "mallionaire",
+            review_body: "Farmyard fun!",
+            review_img_url:
+              "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+            created_at: "2021-01-18T10:00:20.514Z",
+            votes: 2,
+          },
+        });
+      });
+  });
+  test("ERROR 400, responds with the unchanged review when passed no inc_votes", () => {
+    return request(app)
+      .patch("/api/reviews/1")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body).toEqual({
+          errorMessage: "Bad Request - Please provide inc_votes in request",
+          unchangedReview: {
+            review_id: 1,
+            title: "Agricola",
+            category: "euro game",
+            designer: "Uwe Rosenberg",
+            owner: "mallionaire",
+            review_body: "Farmyard fun!",
+            review_img_url:
+              "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+            created_at: "2021-01-18T10:00:20.514Z",
+            votes: 1,
+          },
+        });
+      });
+  });
+  test("ERROR 422: if inc_votes is not a number", () => {
+    const reviewUpdate = { inc_votes: "George" };
+    return request(app)
+      .patch("/api/reviews/1")
+      .send(reviewUpdate)
+      .expect(422)
+      .then(({ body }) => {
+        expect(body.message).toBe(
+          "Unprocessable Entity - inc_votes must be a number"
+        );
+      });
+  });
+  test("ERROR 404: returns review does not exist if review_id does not exist", () => {
+    const reviewUpdate = { inc_votes: 1 };
+    return request(app)
+      .patch("/api/reviews/500")
+      .send(reviewUpdate)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.message).toBe("Bad Request - review_id does not exist");
+      });
+  });
+});
+
```

Comment:
what if you send inc_vote instead of inc_votes? 
Add another error handling test for this :)

Reply:
Isn't this handled in my missing inc_vote test?  
I have a if inc_vote ==== undefined which would catch this error and the message says "Please provide inc_votes in request"

Reply:
Resolved